### PR TITLE
Add jfla ref

### PR DESCRIPTION
**Pull request by Casteran**

### CODE REVIEW

File: doc/hydras.tex [hunk 1]
```diff
@@ -66,15 +66,15 @@
...
+\SetWatermarkText{Work Continuously in Progress}
 \SetWatermarkColor{lightorange}
 
 \author{Pierre Castéran\\ LaBRI, Univ. Bordeaux, CNRS UMR 5800, \thanks{With contributions by Yves Bertot, \'Evelyne Contejean,  Jérémy Damour, Florian Hatat, Pascal Manoury, Karl Palmskog, Clément Pit-Claudel, and Théo Zimmermann. The formalization of primitive recursive functions was originally authored by 
 Russel O'Connor\cite{OConnor05}.}}
 \date{\today}
 %\date{August 2021}
 
-\title{Hydras, Ordinals \& Co.\\ A library in Coq of entertaining formal mathematics}
+\title{Hydras \& Co.\\ A library in Coq of entertaining formal mathematics}
```
Comment (palmskog) [inline doc/hydras.tex:77]:
```suggestion
\title{Hydras \& Co.\\ Formalized mathematics in \coq for inspiration and entertainment}
```

File: doc/hydras.tex [hunk 2]
```diff
@@ -182,10 +182,19 @@ \section{Generalities}
 This document is \emph{not} an introductory text for \coq, and there are many aspects of this proof assistant that are not covered. 
  The reader should already have some basic experience with the \coq system. The Reference Manual and several tutorials are available on the \coq website~\cite{Coq}. The first chapters of textbooks like \emph{Interactive Theorem Proving and Program Development}~\cite{BC04}, \emph{Software Foundations}~\cite{SF} or  \emph{Certified Programming with Dependent Types} ~\cite{chlipalacpdt2011} will give you the right background.
 
-%%%%% ICI
+ 
+ 
+ \subsection{Making and maintaining these  libraries and book}
```
Comment (palmskog) [inline doc/hydras.tex:187]:
```suggestion
 \subsection{Developing and maintaining the \coq code and documentation}
```

File: doc/thebib.bib [hunk 3]
```diff
@@ -827,3 +827,14 @@ @Misc{alectryongithub
   title = 	 {Alectryon},
   howpublished = {\url{https://github.com/cpitclaudel/alectryon}}
 }
+
+
+
+@Misc{jfla2022,
+  author = 	 {Pierre Castéran and Jérémy Damour and Karl Palmskog and Clément Pit-Claudel and Théo Zimmermann},
+  title = 	 {Hydras \& {Co}.: Formalized mathematics in {C}oq for inspiration and entertainment},
+  howpublished = {https://hal.archives-ouvertes.fr/hal-03404668},
```
Comment (palmskog) [inline doc/thebib.bib:836]:
```suggestion
  howpublished = {\url{https://hal.archives-ouvertes.fr/hal-03404668}},
```

File: doc/thebib.bib [hunk 4]
```diff
@@ -827,3 +827,14 @@ @Misc{alectryongithub
...
+  year = 	 {2021},
+  note = 	 {To be published in the proceedings of {JFLA2022}}
```
Comment (palmskog) [inline doc/thebib.bib:838]:
```suggestion
  note = 	 {To be published in the proceedings of {JFLA} 2022}
```

File: doc/hydras.tex [hunk 5]
```diff
@@ -182,10 +182,19 @@ \section{Generalities}
...
+
+ \textbf{TODO} introductory text and link to \cite{jfla2022}
+ 
+ \subsubsection{CI/CD}
```
Comment (palmskog) [inline doc/hydras.tex:191]:
```suggestion
 \subsubsection{Continuous Integration and Continuous Delivery}
```

**Review by palmskog — COMMENTED** [review]
I can add some stuff to replace the TODOs later this week, but I think we can merge this right now to have the structure in place.

**Comment by palmskog** [comment]
@Casteran when you switch to the "Files" tab, there is an option to gather all suggestions into a single commit (which might make CI happier).

**Comment by Casteran** [comment]
@palmskog Oups !

**Review by palmskog — APPROVED** [review]
Looks good to me now, merge when convenient.

**Pull request merged**